src/components/Button/Button.jsx

```diff
@@ -3,50 +3,32 @@ import PropTypes from 'prop-types'
 import './button.css'
 
 /**
- * Primary UI component for user interaction
+ * Component for user interactions that don't include page routing
  */
-const Button = ({ primary, backgroundColor, size, label, ...props }) => {
-  const mode = primary ? 'storybook-button--primary' : 'storybook-button--secondary'
-  return (
-    <button
-      type='button'
-      className={['storybook-button', `storybook-button--${size}`, mode].join(' ')}
-      style={backgroundColor && { backgroundColor }}
-      {...props}
-    >
-      {label}
-    </button>
-  )
-}
+const Button = ({ backgroundColor, size, label, textColor, ...props }) => (
+  <button
+    type='button'
+    className={`button button--${size}`}
+    style={{ backgroundColor, color: textColor }}
```

Comment:
I actually set a default `backgroundColor` prop, so we don't need to check for its existence anymore. also using the `textColor` prop.